.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: run-test
+
+on:
+  push:
+    branches:
+      - main
+      - es6-2020-refactor
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright
+        run: npx playwright install --with-deps
+      - name: build
+        run: npm run build
+      - name: Run your tests
+        run: npm test
```

Comment:
is it worth testing on windows (edge) as well? should be fairly easy with a test matrix https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy

Reply:
done